app/models/mixins/live_metrics_mixin.rb

```diff
@@ -19,7 +24,7 @@ def first_and_last_capture(interval_name = "realtime")
       end.transpose
       adjust_timestamps(firsts, lasts, interval_name)
     rescue => e
-      _log.error("Hawkular service unavailable: #{e.message}")
+      _log.error("LiveMetrics unavailable: #{e.message}")
```

Comment:
Makes sense.  Try to add some detail about the including object in the error message.  Something like:

`"LiveMetrics unavailable for #{self.class.name} id: #{self.id}. #{e.message}"`